minimesos/src/main/java/com/containersol/minimesos/container/ContainerName.java

```diff
@@ -58,8 +58,8 @@ public static boolean belongsToCluster(String containerName, String clusterId) {
     }
 
     /**
-     * @param dockerNames TODO(mwl)
-     * @param clusterId TODO(mwl)
+     * @param dockerNames as returned by <code>container.getNames()</code>
```

Comment:
Use `{@link Container#getNames()}` where `Container` is an existing class. http://stackoverflow.com/questions/5915992/how-to-reference-a-method-in-javadoc